### ISSUE TITLE
Visual Studio 2019 crashes if opening settings

### ISSUE THREAD

**Issue by Sickeroni** [Caphyon/clang-power-tools#805]
# TLDR: 
VS crashes if trying to open Settings

# Long Story
same ;) 

# Versions:
MS Visual Studio Community 2019 - Version 16.5.5 (with .NET Framework Version 4.8.03752) (update of 12.05.2020)
Clang Power Tools 5.9.0

# More information
first time install of Clang Power Tools

# Screenshot
![image](https://user-images.githubusercontent.com/4477853/81744901-21c00080-94a4-11ea-9b83-bd4cb318cae0.png)

# Tested Projects
https://github.com/yuzu-emu/yuzu
https://github.com/yuzu-emu/yuzu-canary

(https://github.com/yuzu-emu/yuzu/wiki/Building-for-Windows used Method 1)

# Extension 
other Parts arent working too

```
--- CLANG TIDY STARTED ---

PROJECT: E:\dev\workspace\repos\yuzu-canary\binory\src\yuzu\yuzu.vcxproj
46: E:\dev\workspace\repos\yuzu-canary\src\yuzu\configuration\configure_audio.cpp
clang-tidy.exe: Unknown command line argument '-quiet'.  Try: 'clang-tidy.exe -help'
clang-tidy.exe: Did you mean '-fix'?

46: E:\dev\workspace\repos\yuzu-canary\src\yuzu\configuration\configure_audio.cpp

--- CLANG TIDY FINISHED ---
```

# Extension 2
possible Duplicate of #804

**Comment by hprica** [comment]
Hello @Sickeroni,

Thank you for taking the time to write your issue in such a detailed manner. 

Clang Power Tools **5.9.1** was just released and it includes the fix for the crash when first opening Settings on a CMake project.

Regarding the issue with Tidy, we are still investigating but we were not able to reproduce it yet. We will update #804 on this matter.

**Comment by Sickeroni** [comment]
still crashes with Clang Power Tools 5.9.1
deinstall and new installing didn't worked either.

**Comment by hprica** [comment]
Could you try to delete Settings.json manually from %appdata%\ClangPowerTools? Let me know if the issue persists.

**Comment by Sickeroni** [comment]
okay strange behaviour

# step 1
removed settings -> not crashing anymore, but still no settingswindow

# step 2
reinstalled -> same behaviour

# step 3
deactivated resharper (installed yesterday after 5.9.1 update) 
-> now it crashes again

**Comment by hprica** [comment]
So after some fiddling around I made the _Nintendo Switch emulator_ project work. Settings seem to work fine. 

I had a couple of crashes when working with just the CMake project and Reshaper installed, without Clang Power Tools. Except for this everything seems OK.

You cannot open Settings from both Toolbar and the context menu? Did you try a different project?

![Capture](https://user-images.githubusercontent.com/28626088/81944575-a24a4280-9605-11ea-89b5-a681d77eb155.PNG)

**Comment by netx123** [comment]
I have the same issue with 7.0.1, I cannot access the settings at all.

**Comment by Enache-Ionut** [comment]
This issue should be fixed in v7.1 (latest version). Please check for updates. 

@netx123 Can you check if the problem is persisting on your machine?

**Comment by netx123** [comment]
@Enache-Ionut I've check 7.1 and it's resolved the issue for me.

**Comment by Enache-Ionut** [comment]
Thank you for the quick testing. I'm glad to hear that. 
I will close this issue for now. If the problem appears again, we'll reopen this thread.
